StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PaymentSheetVerticalViewController.swift

```diff
@@ -54,10 +73,23 @@ class PaymentSheetVerticalViewController: UIViewController, FlowControllerViewCo
         return DynamicHeightContainerView()
     }()
 
+    var rightAccessoryType: RowButton.RightAccessoryButton.AccessoryType? {
+        return RowButton.RightAccessoryButton.getAccessoryButtonType(
+            savedPaymentMethodsCount: savedPaymentMethods.count,
+            isFirstCardCoBranded: savedPaymentMethods.first?.isCoBrandedCard ?? false,
+            isCBCEligible: loadResult.intent.cardBrandChoiceEligible,
+            allowsRemovalOfLastSavedPaymentMethod: configuration.allowsRemovalOfLastSavedPaymentMethod,
+            paymentMethodRemove: configuration.paymentMethodRemove
+        )
+    }
+
     // MARK: - Initializers
 
     init(configuration: PaymentSheet.Configuration, loadResult: PaymentSheetLoader.LoadResult, isFlowController: Bool) {
-        // TODO: Deal with previousPaymentOption
+        // TODO: Deal with previousPaymentOption, default to first saved PM for now
+        if let savedPaymentMethod = loadResult.savedPaymentMethods.first {
+            self.selectedPaymentOption = .saved(paymentMethod: savedPaymentMethod, confirmParams: nil)
```

Comment:
Ah shoot I've implemented `selectedPaymentOption` as well, and in a different way (it's computed)...I'll probably let this PR through and then refactor your implementation / deal with the merge conflicts.